mlflow/server/handlers.py

```diff
@@ -495,6 +507,21 @@ def wrapper(*args, **kwargs):
 ]
 
 
+def _guess_mime_type(file_path):
+    filename = posixpath.basename(pathlib.Path(file_path).as_posix())
+    extension = os.path.splitext(filename)[-1].replace(".", "")
```

Comment:
```suggestion
    extension = pathlib.Path(file_path).suffix.replace(".", "")
```

I haven't tested but this should work.